releases/cockroachcloud-04062020.md

```diff
@@ -0,0 +1,26 @@
+---
+title: What&#39;s New in CockroachCloud
+toc: true
+summary: Additions and changes in CockroachCloud.
+build_for: [cockroachcloud]
+---
+
+## April 06, 2020
+
+In addition to various updates, enhancements, and bug fixes, this beta release includes the following major highlights:
+
+- Free trials of CockroachCloud are now available. [Contact us](mailto:sales@cockroachlabs.com) to request a trial code.
```

Comment:
> `[Contact us](mailto:sales@cockroachlabs.com)`

The contact us link in the next bullet goes to a form. Do we want this to link to a form as well? (maybe https://www.cockroachlabs.com/contact-sales/)

(I don't have insight into the preference of the Sales team. Mostly thinking aloud from a consistency standpoint).

Reply:
Oh, I didn't know about the form. I think that's a better method. Correcting now.